### PR TITLE
Feat: update grid and breakpoints

### PR DESCRIPTION
**Pull request by mjmonline**

### CODE REVIEW

File: src/theme/Theme.stories.tsx [hunk 1]
```diff
@@ -71,9 +72,9 @@ storiesOf('Theme', module)
   })
   .add('Screen sizes', () => (
     <Display
-      items={Object.entries(theme.size).map(([title, size]) => ({
+      items={Object.entries(theme.breakpoints).map(([title, breakpoint]) => ({
```
Comment (shmidt-i) [inline src/theme/Theme.stories.tsx]: here I think we can display both size and offset

File: src/theme/theme.ts [hunk 2]
```diff
@@ -45,17 +46,50 @@ export const rawColor = {
...
+    size: 1440,
+    offset: 24,
+  },
+  xl: {
+    size: 1680,
+    offset: 10,
+  },
+};
+
+const size: Theme['size'] = {
```
Comment (shmidt-i) [inline src/theme/theme.ts]: Let's write `deprecated` helper and use it here

File: src/theme/theme.types.ts [hunk 3]
```diff
@@ -130,22 +145,20 @@ export type Theme = {
...
-`
+     * styled.div`${({ theme }) => theme.media.lessThan(theme.breakpoints.md)} {}`
      */
-    lessThan: (/** One of theme.size */ size: number) => MediaQuery;
-    greaterThan: (/** One of theme.size */ size: number) => MediaQuery;
-    between: (
-      /** One of theme.size */ size1: number,
-      /** One of theme.size */ size2: number,
-    ) => MediaQuery;
+    lessThan: (number: NumberOrObjectWithNumber) => MediaQuery;
```
Comment (shmidt-i) [inline src/theme/theme.types.ts]:
```suggestion
    lessThan: (size: NumberOrObjectWithNumber) => MediaQuery;
```

File: src/theme/theme.types.ts [hunk 4]
```diff
@@ -130,22 +145,20 @@ export type Theme = {
...
+    greaterThan: (number: NumberOrObjectWithNumber) => MediaQuery;
```
Comment (shmidt-i) [inline src/theme/theme.types.ts]:
```suggestion
    /**
     * @param size One of theme.breakpoints
     * @example
     * styled.div`${({ theme }) => theme.media.greaterThan(theme.breakpoints.md)} {}`
     */
    greaterThan: (size: NumberOrObjectWithNumber) => MediaQuery;
```

File: src/theme/theme.ts (hunk 2 above)
Comment (shmidt-i) [reply src/theme/theme.ts]:
```js
const deprecate = message => target => {
  if (process.env.NODE_ENV !== "production") {
    return typeof Proxy === "undefined"
      ? target
      : new Proxy(target, {
          get: function(getTarget, getProp) {
            console.warn(`Deprecated: ${message}`);
            return getTarget[getProp];
          },
          apply: function(applyTarget, thisArg, argumentsList) {
            console.warn(`Deprecated: ${message}`);
            return applyTarget.apply(thisArg, argumentsList);
          }
        });
  }
  return target;
};

```

Comment (shmidt-i) [reply src/theme/theme.ts]: Something like that

File: src/theme/Theme.stories.tsx (hunk 1 above)
Comment (mjmonline) [reply src/theme/Theme.stories.tsx]: fixed

File: src/theme/theme.ts (hunk 2 above)
Comment (mjmonline) [reply src/theme/theme.ts]: fixed

File: src/theme/theme.types.ts (hunk 3 above)
Comment (mjmonline) [reply src/theme/theme.types.ts]: fixed

File: src/theme/theme.types.ts (hunk 4 above)
Comment (mjmonline) [reply src/theme/theme.types.ts]: fixed

**Pull request merged**